client/src/pages/workspace/addBoard/AddBoard.tsx

```diff
@@ -26,13 +26,15 @@ const AddBoard: React.FC<AddBoardProp> = ({addBoard}: AddBoardProp) => {
         }
     }
 
+    const textFieldInputStyle = {
+        padding: "0.4em"
+    };
```

Comment:
don't create variables in the component. You should create it in a state or outside of the component. The problem is that your variable will be recreated for each rerender operation. You can check it if you write console.log(textFieldInputStyle) and do some actions